docs/sphinx/source/whatsnew/v0.11.1.rst

```diff
@@ -44,15 +44,14 @@ Documentation
 * Added gallery example demonstrating the application of
   several spectral mismatch factor models.
   (:issue:`2107`, :pull:`2114`)
-
 * Added gallery example on calculating cell temperature for
   floating PV. (:pull:`2110`)
-
 * Added gallery example demonstrating how to use
   different Perez coefficients in a ModelChain.
   (:issue:`2127`, :pull:`2148`)
-
 * Removed unused "times" input from dni_et() function (:issue:`2105`)
```

Comment:
I noticed this whatsnew entry is missing a link to its PR (#2156) so should I add that in as part of this PR? The tag was also "testing", so I am wondering whether this entry for #2156/#2105 should be moved from documentation to testing. Besides that, I think I have addressed all reviewers' suggestions.

Reply:
> be moved from documentation to testing

yes but not in this PR

Reply:
Is it possible to update it in the original merged PR? If not (or not advised) I can just open a new one.

Reply:
AFAIK it is not possible to edit a merged PR (they can be reverted but not changed). A new PR, or a maintainer can do it directly in the whatsnew file.

Reply:
We usually just take care of this stuff all at once as part of the "finalize release" PR.  That PR will be editing the whatsnew file anyway (to set the release date if nothing else).  Not worth worrying about cleaning up the whatsnew file until then IMHO.